docs/CodeTransformation.md

```diff
@@ -110,6 +110,14 @@ interface TransformOptions<OptionType> {
   // names are copied from babel: https://babeljs.io/docs/en/options#caller
   supportsDynamicImport: boolean;
   supportsExportNamespaceFrom: boolean;
+  /**
+   * The value is:
+   * - `false` if Jest runs without Node ESM flag `--experimental-vm-modules`
+   * - `true` if the file extension is defined in [extensionsToTreatAsEsm](Configuration.md#extensionstotreatasesm-arraystring)
+   * and Jest runs with Node ESM flag `--experimental-vm-modules`
+   *
+   * See more at https://jestjs.io/docs/next/ecmascript-modules
```

Comment:
These would then be wrong when rolling a new version of the docs. Are we able to make docusaurus generate this link (i.e. use relative link within the source code)?

/cc @slorber